pylint_django/tests/input/migrations/0004_with_backwards.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=missing-docstring
+from django.db import migrations
+
+
+def forwards_test(apps, schema_editor):
+    pass
+
+
+def backwards_test(apps, schema_editor):
+    pass
+
+
+class Migration(migrations.Migration):
+
+    operations = [
+        migrations.RunPython(forwards_test, backwards_test)
+    ]
```

Comment:
see `test_db_performance_plugin()` inside `pylint_django/tests/test_func.py` for hints. If you want to experiment as-is you will have to duplicate the test function b/c it needs the additional file paths. Migrations are automatically excluded by the main pylint-django plugin, see plugin.py -> load_configuration() -> linter.config.black_list

Reply:
Will wait to see which structure for this checker will be used.